package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "reality-capture-main",
+  "version": "1.0.0",
+  "description": "This project contains iTwin API SDK for ContextCapture and RealityDataAnalysis as well as reality data API utils, in typescript and python.",
```

```suggestion
  "description": "This project contains iTwin API SDK for ContextCapture and RealityDataAnalysis as well as reality data API utils in typescript.",
```

pkg.json is only relevant for the ts api

---

fixed